sdk-src/src/test/java/com/hpe/adm/nga/sdk/model/TestEntityUtil.java

```diff
@@ -0,0 +1,142 @@
+package com.hpe.adm.nga.sdk.model;
+
+import org.junit.Test;
+
+import java.time.ZonedDateTime;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class TestEntityUtil {
+
+    @Test
+    @SuppressWarnings("rawtypes")
+    public void testAreFieldModelsEqualByContent() {
+
+        FieldModel fieldModelOne;
+        FieldModel fieldModelTwo;
+
+        fieldModelOne = new LongFieldModel("id", 1L);
+        fieldModelTwo = new StringFieldModel("id", "1");
+        assertFalse(EntityUtil.areFieldModelsEqualByContent(fieldModelOne, fieldModelTwo));
```

Comment:
@atothhpe Usually each of these cases should be an individual test. That way, for a single run of the tests you can determine the status of the tests. The way they are written now, if this first assert fails, you won't know the status of the cases below until you fix this failing cases